Dockerfile

```diff
@@ -14,5 +14,5 @@ RUN apk --update add go git\
   && apk del go git \
   && rm -rf $GOPATH /var/cache/apk/*
 
-CMD /coco-fleet-unit-healthcheck -fleetEndpoint=$FLEET_ENDPOINT -socks-proxy=$SOCKS_PROXY -timerBasedServices=deployer.service,image-cleaner.service,tunnel-register.service
+CMD /coco-fleet-unit-healthcheck -fleetEndpoint=$FLEET_ENDPOINT -socks-proxy=$SOCKS_PROXY -timerBasedServices=mongo-backup.service,deployer.service,image-cleaner.service,tunnel-register.service
```

Comment:
Should this instead be read from etcd, and services can just say they want to be whitelisted by registering a value in etcd, something like `/ft/config/healthcheck/whitelist`?

Can be a seperate issue/PR, we can merge this (+1)